doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,216 @@
+.. _detailed-release-procedure:
+
+Release: recipes runs and comparison
+====================================
+
+The release procedure for ESMValTool is a fairly involved process (at the moment), so it
+is important to be very well organized and to have documented each procedural steps, so that
+the next release manager can follow said steps, and finalize the release without any delays.
+
+The workflow below assumes an ESMValCore release candidate, or a completed stable release, have been released
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on `DKRZ/Levante
+<https://docs.dkrz.de/>`_.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in the ``main`` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+- path to the output directories on DKRZ/Levante
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented: make a copy of the environment file,
+and attach it in the release testing issue; to record the environment in a yaml file use e.g.
+
+.. code-block:: bash
+
+  conda env export > ToolEnv_v2xx_Test.txt
+
+Modifications to configuration files need to be documented as well.
+To test recipes, it is recommended to only use the default options and DKRZ data directories, simply by uncommenting
+the DKRZ-Levante block of a newly generated ``config-user.yml`` file.
+
+Submit run scripts - test recipe runs
+-------------------------------------
+
+We are now ready to start running all the available recipes, to compare output against previous release. Running is currently done
+via batch scripts submitted to a schedulers (SLURM). Generate the submission scripts using the ``generate.py`` `Python script
+<https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/utils/batch-jobs/generate.py>`_. 
```

Comment:
It would be nicer to link to the documentation describing this script instead of to the actual code